compiler/crates/relay-codegen/src/printer.rs

```diff
@@ -27,16 +28,18 @@ pub fn print_operation(
     schema: &SDLSchema,
     operation: &OperationDefinition,
     js_module_format: JsModuleFormat,
+    enable_reduced_metadata: &FeatureFlag,
 ) -> String {
-    Printer::without_dedupe(js_module_format).print_operation(schema, operation)
+    Printer::without_dedupe(js_module_format).print_operation(schema, operation, enable_reduced_metadata)
```

Comment:
can we add this flag to the `Printer` itself, so we do not pass it around all methods?